pg_bm25/Cargo.toml

```diff
@@ -19,10 +19,14 @@ pg_test = []
 telemetry = ["shared/telemetry"]
```

Comment:
Remember to add instructions to add `pg_bm25` to `shared_preload_libraries` in the README